images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -318,7 +318,8 @@ function Get-VSExtensionVersion
 
 function Get-ToolsetContent
 {
-    $toolsetJson = Get-Content -Path $env:TOOLSET_JSON_PATH -Raw
+    $toolsetPath = Join-Path $env:TEMP "toolset.json"
+    $toolsetJson = Get-Content -Path $toolsetPath -Raw
```

Comment:
Wouldn't this be better?
```suggestion
    $toolsetJson = Get-Content -Path (Join-Path $env:TEMP "toolset.json") -Raw
```

Reply:
Thank you for your comment, I would prefer to keep it as multiple line because in my opinion, it is better for readability.